keepassxc-browser/background/page.js

```diff
@@ -10,6 +10,7 @@ const defaultSettings = {
     autoRetrieveCredentials: true,
     showNotifications: true,
     showLoginNotifications: true,
+    showUsernameIcons: true,
```

Comment:
I'm on the fence for this variable name, I feel like it should be `showLoginFormIcon` (not plural).